config/routes.rb

```diff
@@ -1,4 +1,6 @@
-Rails.application.routes.draw do  
+Rails.application.routes.draw do
+  post "/graphql", to: "graphql#execute"
+
   namespace :api, defaults: { format: 'json' } do
     get '/bikeway_networks', to: 'bikeway_networks#nearest_network'
```

Comment:
More as a TODO or just something to keep in mind (and honestly at this point you have more graphql experience than I do so I can be wrong): my understanding is now that we have graphql implemented, we can refactor our controller actions to use graphql under the hood, but still return the data in our specified format. So the obvious case would be in `bikeway_networks_controller.rb`, we can now replace `nearest_network` action with the equivalent graphql call. The benefit here would be that it would be easier to maintain in the future. Not needed for this pr or anything.